diagnostic.md

```diff
@@ -105,13 +152,38 @@ JSON.
 ```
 
 ```sh
-# your answer here
+# this is my comment
+curl --request PATCH http://10.13.105.206:3000/items/1 \
+ --header "Content-Type: application/json" \
+ --data '{
+   "done": true
+ }'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function(event) {
+  event.preventDefault();
+  // data stores here
+  let itemIdInput = $(event.target).find('[name="name_id"]');
+  // pull the value out // .val() get value
+  //let itemId = $(event.target).find('[name="item_id"]').val(); // event.target is the form, access jquery method. [name=], pseudo id selector
+  let item = new FormData(event.target);
+  $.ajax({
+    url: 'http://localhost:3000/items' + itemId, // make this a variable
+    type: 'PATCH',
+    contentType: false,
+    processData: false,
+    data: item
+  })
+  .done(logResponseBody)
+  .fail(logRequestError);
+  // .val("") set value
+  //$(event.target).find('[name="item_id"]').val("");
+
+  itemIdInput.val('');
+};
```

Comment:
Oh... this is actually a correct answer for Q12, not Q11. I will accept it as your Q12 answer, though.